actions.php

```diff
@@ -13,15 +13,15 @@
     "email_operator_group"      => "Stuur email aan operatorgroep",
     "email_user"                => "Stuur email aan gebruiker",
     "email_user_group"          => "Stuur email aan gebruikersgroep",
-    "unassign_operator"         => "Verwijder toewijzing van ticket",
+    "unassign_operator"         => "Verwijder operator van ticket",
     "update_department"         => "Update afdeling",
     "update_priority"           => "Update prioriteit",
     "update_status"             => "Update status",
-    
+
     /*
      * 2.0.3
      */
-    "update_custom_field"       => "Update custom field van ticket",
+    "update_custom_field"       => "Update aangepast veld van ticket",
 
     /*
      * 2.2.0
```

Comment:
This file is missing the 2.3.0 to 2.6.0 translations

Reply:
added the missing strings and finalized translation for this file